### ISSUE TITLE
Why MERT-v1-330M was trained with `feat_extract_norm="group"` instead of "layer"?

### ISSUE THREAD

**Issue by LiableFishYS** [yizhilll/MERT#19]
Hey! Thank you for your amazing work!

I've noticed that while the configuration for large MERT-v1-330M matches e.g. [wav2vec2-large-robust](https://huggingface.co/facebook/wav2vec2-large-robust/blob/main/config.json) in terms of transformer encoder parameters, you used `feat_extract_norm="group"` instead of `feat_extract_norm="layer"`. Wav2Vec2 authors [mentioned](https://github.com/facebookresearch/fairseq/issues/3277#issuecomment-788177468) that using `layer` provides more robust training for larger models.

Also, `layer` computes statistics for each time step independently which solves issue with different model outputs for padded and unpadded inputs (unlike with `group` where statistics are compute for each channel across all time steps including padding ones since there is no `attention_mask`-like mechanism in feature extractor). Which is huge benefit because it allows running inference in batches "properly".

Any reasons why you preferred `group` other `layer`?

**Comment by Skrill2001** [comment]
Hi, I have a similar question. So does this mean that the current MERT cannot perform correct batch inference with padding, even when an attention_mask is provided to the model? In my tests, I also noticed that the results differ when padding and attention_mask are used compared to when they are not.
